docs/getting_started.md

```diff
@@ -31,17 +34,57 @@ if (!("SingleCellExperiment" %in% installed.packages())) {
 
 library(SingleCellExperiment)
 # read in the RDS file, including the path to the file's location
-sce <- readRDS("SCPCS000000/SCPCL000000_filtered.rds")
+processed_sce <- readRDS("SCPCS000000/SCPCL000000_processed.rds")
 ```
 
 More resources for learning about `SingleCellExperiment` objects:
 
 - [`SingleCellExperiment` objects Vignette](https://bioconductor.org/packages/devel/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html) on working with `SingleCellExperiment` objects
 - [Orchestrating Single Cell Analysis chapter on the `SingleCellExperiment` class](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html)
 
-## Quality control
+## Working with the processed `SingleCellExperiment` objects
+
+The `processed` objects contain both the raw and normalized gene expression data, so many of the steps described in the section below can be skipped.
```

Comment:
This sentence seems oddly placed to me. I think you are referring to the "working with filtered" section, but that isn't really obvious. This can probably be removed with the link to the processing description moved up?